### PR TITLE
terraform set up and parameters added to all environments

### PR DESCRIPTION
**Pull request by humulla**
## Link to JIRA ticket

[https://hackney.atlassian.net/jira/software/projects/PA/boards/35?selectedIssue=PA-361](https://hackney.atlassian.net/jira/software/projects/PA/boards/35?selectedIssue=PA-361)

## Describe this PR

Sets up the terraform for creating a Postgres instance in AWS for staging and production only.

Checklist
- [x] Code pipeline builds correctly

### _Follow up actions after merging PR_

- Add terraform setup for DMS.
- Check deployment to staging and production works

### CODE REVIEW

File: terraform/staging/main.tf [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+}
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+
+locals {
+  parameter_store = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter"
+}
+
+data "aws_iam_role" "ec2_container_service_role" {
```
Comment (elcorbs) [inline terraform/staging/main.tf:22]: this isn't needed

File: terraform/staging/main.tf [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  name = "ecsServiceRole"
+}
+
+data "aws_iam_role" "ecs_task_execution_role" {
```
Comment (elcorbs) [inline terraform/staging/main.tf:26]: Neither is this

**Review by elcorbs — APPROVED** [review]
Looks good, have all of the parameters mentioned here been added to parameter store already?

**Comment by humulla** [comment]
> Looks good, have all of the parameters mentioned here been added to parameter store already?

Yep the parameters have been created in staging and prod

**Review by LBHMGeorgieva — APPROVED** [review]

**Pull request merged**